ols/utils/config.py

```diff
@@ -24,11 +23,9 @@ def init_empty_config() -> None:
     global config
     global ols_config
     global llm_config
-    global dev_config
```

Comment:
intent is to do subsequent separate PRs to remove the other globals once this PR/approach is agreed on?

Reply:
Though so. I started with other globals in the meantime and it would take many PRs - with the right order - to make the change (prolong the effort significantly). So I did [943](https://github.com/openshift/lightspeed-service/pull/943) instead.